src/map/utils/battleutils.cpp

```diff
@@ -4236,6 +4238,8 @@ namespace battleutils
         resist = 1.0f + ( floor( 256.0f * ( PDefender->getMod(Mod::DMGPHYS) / 100.0f ) ) / 256.0f )
                       + ( floor( 256.0f * ( PDefender->getMod(Mod::DMG)     / 100.0f ) ) / 256.0f );
         resist = dsp_cap(resist, 0.5f, 1.5f); //assuming if its floored at .5f its capped at 1.5f but who's stacking +dmgtaken equip anyway???
+        resist = resist + (floor(256.0f * (PDefender->getMod(Mod::DMGPHYS_II) / 100.0f)) / 256.0f);
```

Comment:
physical damage taken II is additive, not multiplicative with physical damage taken